src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/advice/UserProfileConstants.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.reform.userprofileapi.controller.advice;
+
+public class UserProfileConstants {
+
+    private UserProfileConstants() {
+    }
+
+    public static final String NAME_FORMAT_REGEX = "^[A-Za-z'-]+$";
+}
```

Comment:
We do not currently have a Constants class for User Profile - _only an Error Constants Class_ 
So I have created this to store any future Constants we may create.